### ISSUE TITLE
Pass kwargs to internal solvers

### ISSUE THREAD

**Issue by abelsiqueira** [JuliaSmoothOptimizers/JSOSuite.jl#17]
For instance
```
function quadprog(Q :: [...]; kwargs...)
  [...]
  ipopt(nlp, print_level=0; kwargs...)
end
```

**Comment by abelsiqueira** [comment]
cf @RenanOD

**Comment by RenanOD** [comment]
This was solved by PR #19 and should be closed

**Comment by abelsiqueira** [comment]
Thanks
